### ISSUE TITLE
digest "Problems and Strategies in Financing Voluntary Free Software Projects"

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#179]
"[Problems and Strategies in Financing Voluntary Free Software Projects](http://mako.cc/writing/funding_volunteers/funding_volunteers.html)"

**Comment by whit537** [comment]
Learned about this underneath https://github.com/gratipay/gratipay.com/issues/3319.

**Comment by whit537** [comment]
https://twitter.com/whit537/status/585885586302619648

**Comment by whit537** [comment]
Heh. We're the second and fourth hit on Google. :-)

![screen shot 2015-07-17 at 3 55 14 pm](https://cloud.githubusercontent.com/assets/134455/8755907/51c1dfa8-2c9c-11e5-82f0-1a9669b9876e.png)

**Comment by whit537** [comment]
Here's [a patch that fixes typos in v0.2.1]
(https://gist.github.com/whit537/28ae767bb230bc8621fc) of "Problems and Strategies."

**Comment by whit537** [comment]
Hill starts by describing the central role that voluntary labor plays in open-source software, and its benefits: high quality, low cost, and institutional independence. The drawback of voluntary labor is that it can't be scheduled as predictably as paid labor. Introducing paid labor into a volunteer project, however, brings problems: "crowding out" of volunteers, decreased transparency, and lower quality. Mako identifies seven tactics for funding volunteer projects. Here they are (lightly edited) in order of how much risk they hold for introducing the problems identified:

| n | Strategy | Examples | Notes |
|---|:-----|:--------|:----------|
| 1 | Refuse all funding. | | |
| 2 | Fund shared infrastructure. | hardware, bandwidth | |
| 3 | Fund "capacity." | an accounting system | |
| 4 | Fund events. | conferences, sprints | Watch out for favoritism. |
| 5 | Fund non-technical work. | project management, book-keeping, grant writing | Relates to (3), funding capacity. |
| 6 | Fund difficult work. | technically challenging, tedious, time-sensitive, and/or high-demand projects | Make sure there is community buy-in to avoid crowding out. |
| 7 | Accept work product from corporations. |  | Maintain diverse corporate relationships to ensure independence. |

The piece ends with advice on maintaining transparency: report everything back to a public, open-registration mailing list (~= GitHub).

**Comment by whit537** [comment]
How does this relate to Gratipay? What's been our experience with funding per Hill's schema? How does our payroll model relate to his schema?

**Comment by whit537** [comment]
| n | Strategy | Gratipay's Experience |
|---|:-----|:--------|
| 1 | Refuse all funding. | We've [turned down](https://github.com/gratipay/inside.gratipay.com/issues/68) overtures for all *venture* funding, at least. |
| 2 | Fund shared infrastructure. | GitHub, Travis CI, Transifex, etc. |
| 3 | Fund "capacity." | I suppose the rest of our operations budget fits here. |
| 4 | Fund events. | We paid for the first retreat individually, but for the second we [tapped](https://github.com/gratipay/inside.gratipay.com/issues/124) into Gratipay funds. We also [paid some](https://github.com/gratipay/inside.gratipay.com/issues/111#issuecomment-90318507) for PyCon. |
| 5 | Fund non-technical work. | We burst the dam on this with a [lawyer](https://github.com/gratipay/inside.gratipay.com/issues/282). Now we're looking at engaging a [CPA](https://github.com/gratipay/inside.gratipay.com/issues/285) and a  [designer](https://github.com/gratipay/inside.gratipay.com/issues/298). |
| 6 | Fund difficult work. | This is where our payroll model comes in. |
| 7 | Accept work product from corporations. | Balanced gave us a PR [integrating us with them](https://github.com/gratipay/gratipay.com/pull/137), as well as our [second design](https://github.com/gratipay/gratipay.com/issues/66). |

**Comment by whit537** [comment]
Our payroll model, wherein everyone sets their own take, is a new way to fund volunteer work that didn't exist when Hill wrote his article. Our response, I think, is in fact that our model solves the problem of funding core work in a volunteer project, by giving each individual full responsibility for their own guilt and resentment. We remove the artificial technical and bureaucratic limitations on the proper functioning of a team as a socioeconomic unit.

The [bounty](https://www.bountysource.com/) [model](https://freedomsponsors.org/) is the other new development since 2005, when "Problems and Strategies" was published, and even since 2012, when it was last updated. Gratipay's critique of the bounty model is that it encourages competition rather than collaboration (cf. @rohitpaulk's [report](https://botbot.me/freenode/gratipay/2015-02-19/?msg=32363111&page=1) re: Assembly).

**Comment by whit537** [comment]
I think Hill's seven-fold schema is helpful. Let's refer back to it when we're faced with decisions about how to spend money ([e.g.](https://github.com/gratipay/inside.gratipay.com/issues/298)).

**Comment by whit537** [comment]
Anyone else want to jump in before I close this ticket?

**Comment by tshepang** [comment]
nothing new (to me) here, so I'm kool

**Comment by tshepang** [comment]
I'm impressed by how well the Gratipay sort of funding works (each worker takes what they feel they deserve, what you call guilt and resentment), though it needs to see wider use to get truly validated.

**Comment by tshepang** [comment]
It's the same as envisioned by Anarchists I believe... the ideal Anarchist economy. Exciting... I really hope it works.

**Comment by whit537** [comment]
> I'm impressed by how well the Gratipay sort of funding works

Yeah! A pleasant surprise. We discovered something interesting! I don't know enough about Anarchism to say whether that label applies. To me, the reality is more interesting than the label, anyway. :-)

**Comment by whit537** [comment]
@SimonSarazin I see that you engaged with "Problems and Solutions" a bit on http://unisson.co/fr/wiki/retribuer/. Your point about labor law at the bottom of that post is germane: we're wrestling with that over on #242, and it's a big blocker to our bringing back our unique payroll distribution feature, which we turned off in Gratipay 2.0 precisely because of labor law concerns.

I tried to visit http://cobudget.co/, but my library won't let me visit the Netherlands. :-(

![screen shot 2015-08-10 at 6 04 02 pm](https://cloud.githubusercontent.com/assets/134455/9186194/83d1cec0-3f90-11e5-8a40-1155033dcbb9.png)

Also, where is the quote at the top of http://simons.fr/2013/11/gittip/ from?
